### PR TITLE
Get latest

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Also, add comment about saveAsync to itemId example (related PR https://github.com/OfficeDev/office-js-docs-pr/pull/1488).

### CODE REVIEW

File: docs/code-snippets/outlook-snippets.yaml [hunk 1]
```diff
@@ -586,6 +586,8 @@ Office.AppointmentRead#itemId:member:
     // identifier. If the `itemId` property returns `null` or
     // `undefined`, it saves the item to the store and gets the
     // item identifier from the asynchronous result.
+    // Important: saveAsync was introduced with requirement set 1.3
```
Comment (AlexJerabek) [inline docs/code-snippets/outlook-snippets.yaml]: There's a bit of inconsistency with how code ticks are used in this comment. `itemId` should either always be surrounded or never.

Comment (AlexJerabek) [inline docs/code-snippets/outlook-snippets.yaml]: Nit: consider bolding Important (**)

**Comment by o365devx** [comment]
Docs Build status updates of commit _[532c6cf](https://github.com/OfficeDev/office-js-docs-reference/commits/532c6cf27886d482ac61c0a611518ea00792fab0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office/office.document.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office/office.document.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office/office.filetype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office/office.filetype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office/office.slice.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office/office.slice.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_4/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_4/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_4/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_4/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_5/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_5/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_5/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_5/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_6/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_6/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_6/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_6/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_7/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_7/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_7/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_7/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_8/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_8/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_8/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_8/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-office/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/api-extractor-inputs-office/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/json/office/office.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/office/office.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_1/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_1/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_2/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_2/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_3/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_3/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_4/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_4/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_5/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_5/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_6/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_6/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_7/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_7/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_8/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_8/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/script-inputs/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/office_preview.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/script-inputs/office_preview.d.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C10%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201912102235214773-575%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Lw%2BOBscIG32QmV5vJ6OSq8pxOg5JLrdRtPvILcTheyg%3D&st=2019-12-10T22%3A56%3A43Z&se=2020-01-10T23%3A01%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[83d3bc9](https://github.com/OfficeDev/office-js-docs-reference/commits/83d3bc95dc7badcb40fd75cf44b8e9ce8b1de9f1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office/office.document.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office/office.document.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office/office.filetype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office/office.filetype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office/office.slice.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office/office.slice.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_4/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_4/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_4/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_4/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_5/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_5/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_5/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_5/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_6/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_6/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_6/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_6/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_7/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_7/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_7/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_7/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_8/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_8/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_8/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_8/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-office/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/api-extractor-inputs-office/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/json/office/office.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/office/office.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_1/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_1/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_2/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_2/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_3/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_3/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_4/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_4/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_5/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_5/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_6/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_6/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_7/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_7/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook_1_8/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/outlook_1_8/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/script-inputs/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/office_preview.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/generate-docs/script-inputs/office_preview.d.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C10%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201912102246163215-575%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3sqVuFFU3bBlYFPZ6B0bd9RE%2F6X8B2WWJ6su0X%2BtnUE%3D&st=2019-12-10T23%3A23%3A10Z&se=2020-01-10T23%3A28%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**